nitrates/analysis_seeds/UpperLimits.py

```diff
@@ -282,11 +215,23 @@ def main(args):
                 ul_5sigma_new.append(flux_upper_limit_new)
             
             filename = f"ul_5sigma_{spec_temp}_{dur_values[k]}.txt"
```

Comment:
Can you change this from .txt to .csv?

Reply:
done

Reply:
Hm I don't see any new commits. The last one I see is from 9 hours ago

Reply:
just pushed commits